modules/apps/foundation/friendly-url/friendly-url-service/src/main/java/com/liferay/friendly/url/servlet/FriendlyURLServlet.java

```diff
@@ -202,30 +203,34 @@ else if (_log.isWarnEnabled()) {
 		Map<String, String[]> params = request.getParameterMap();
 
 		try {
-			LayoutFriendlyURLComposite layoutFriendlyURLComposite =
-				portal.getLayoutFriendlyURLComposite(
-					group.getGroupId(), _private, friendlyURL, params,
-					requestContext);
+			LayoutFriendlyURLSeparatorComposite
+				layoutFriendlyURLSeparatorComposite =
```

Comment:
Hey Marco,

If we do it this way, we cannot backport this because it would break the code of any 3rd party that has created the now deprecated LayoutFriendlyURLComposite. To be able to backport it now we need to support both for the time being. Then for 7.1 we drop support for the deprecated one.

What do you think?